### ISSUE TITLE
Página da armadilha

### ISSUE THREAD

**Issue by vgeorge** [aetrapp/webapp#28]
Exibir foto da armadilha na posição original (retrato ou paisagem)

Enxugar data e hora para o formato “02/03/2018 - 16:20"

Falta gráfico com série histórica e média para referência

**Comment by miguelpeixe** [comment]
- Foto não tem sua posição manipulada no frontend
 - Data ok
 - Gráfico com série histórica existe, basta subir mais de uma amostra para uma armadilha (não é exibido gráfico em caso de apenas 1 amostra propositalmente)
 - Para a média basta somar e dividir? cc @vgeorge

**Comment by vgeorge** [comment]
No gráfico o Oda se refere à média de contagens da cidade, como neste gráfico:

![grafico-medias](https://user-images.githubusercontent.com/329694/37058286-f2bae3fa-2181-11e8-8f72-abb037fdf38d.jpeg)

O serviço de cidades inclui a média de contagens por semana.

**Comment by miguelpeixe** [comment]
isso aqui?
 https://github.com/aetrapp/aetrapp/commit/91105f0d6549f51196ad1fe776ac599621dd798b

**Comment by vgeorge** [comment]
Sim. Se você dar um get em uma cidade com amostras válidas vai gerar algo do tipo:

```
{
    "id": "2611606",
    "stateId": "PE",
    "name": "Recife",
    "isCapital": true,
    "point": {
        "type": "Point",
        "coordinates": [-34.8889419446, -8.0627624831]
    },
    "withoutAccents": "Recife",
    "alternativeNames": null,
    "microregion": "Recife",
    "mesoregion": "Metropolitana de Recife",
    "eggMeanSeries": {
        "2018-8": 3,
        "2018-9": 5
    }
}
```

**Comment by vgeorge** [comment]
Recapitulando o que falta aqui:

- [ ] média da cidade no gráfico;
- [ ] link para o dono da armadilha.

**Comment by vgeorge** [comment]
Na última versão da API, o histórico da média de contagens é retornado no objeto da cidade e é preciso atualizar o webapp para refletir isso. Também falta o link para o dono da armadilha.

**Comment by odascatolini** [comment]
Falta exibir foto da armadilha na posição original (retrato ou paisagem)

**Comment by vgeorge** [comment]
@odascatolini agora o aplicativo não altera a orientação original da foto, por favor, verifique se está ok enviando novas fotos para armadilhas e amostras.

**Comment by vgeorge** [comment]
@miguelpeixe O webapp está sempre colocando as fotos de armadilhas na horizontal, mesmo que elas sejam na vertical. 

Sobre o mobile há uma issue: https://github.com/aetrapp/mobile/issues/56

**Comment by miguelpeixe** [comment]
Não consegui reproduzir:

![screen shot 2018-04-22 at 15 31 09](https://user-images.githubusercontent.com/820752/39098504-3bef7df8-4642-11e8-8a35-fecf8348b7a2.png)

**Comment by vgeorge** [comment]
Pois é, depois eu revisei novamente e acho que foi confusão minha, colocando para revisão.

**Comment by vgeorge** [comment]
Reabrindo, apareceu um caso onde a foto original está na vertical, mas no painel mostra horizontal:

http://painel.aetrapp.org/traps/8DT9F8JWUKG37

**Comment by odascatolini** [comment]
Para reproduzir cadastrem uma nova armadilha (cadastrem em Joao Pessoa e depois deletem por favor), enviem uma foto na posição retrato e vejam como fica no webapp

**Comment by miguelpeixe** [comment]
Este problema segue **não sendo do webapp**, não há manipulação da rotação da imagem.

**Comment by miguelpeixe** [comment]
A biblioteca [JIMP](https://github.com/oliver-moran/jimp) pode ser uma solução permanente de backend, utilizando o método `exifOrientation()` para armazenar a imagem com a orientação correta.
